apps/server/src/server.ts

```diff
@@ -12,6 +13,17 @@ import userRoutes from './routes/user';
 
 const app = express();
 app.use(express.json());
+app.get('/', (req, res, next) => {
+  res.setHeader('Access-Control-Allow-Origin', '*');
+  res.setHeader('Access-Control-Allow-Methods', 'POST, GET, PUT');
+  res.setHeader('Access-Control-Allow-Headers', 'Content-Type');
+  next();
+});
+const corsOptions = {
+  origin: 'http://localhost:3000',
+  credentials: true,
+};
+app.use(cors(corsOptions));
```

Comment:
I think we should allow all CORS Requests

```suggestion
app.use(cors())
```